src/components/site-card/logs-viewer.tsx

```diff
@@ -0,0 +1,42 @@
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+import { RawLogs, StyledPanel, StyledPanelHeader } from "gatsby-interface"
+import AnsiParser from "ansi-to-html"
+import { useMemo } from "react"
+
+export interface IProps {
+  logs: Array<string>
+  onDismiss: () => void
+}
+
+export function LogsViewer({ logs, onDismiss }: IProps): JSX.Element {
+  const parser = useMemo(
+    () =>
+      new AnsiParser({
+        stream: true,
+      }),
+    []
+  )
+
+  return (
+    <StyledPanel>
+      <StyledPanelHeader onCloseButtonClick={onDismiss}>Logs</StyledPanelHeader>
+      <RawLogs
+        // RawLogs expect "message" to be a string but in factthey also support elements
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
So this is a workaround for bad typings in gatsby-interface? Can this be fixed there?